### DISCUSSION TITLE
Low power mode indication in menu bar icon

### DISCUSSION THREAD

**Discussion by mr-kew** [AppHouseKitchen/AlDente-Battery_Care_and_Monitoring#1220] [Ideas]
Hello, I would like to use the right click menu bar icon to enable/disable low power mode, but I find it pretty confusing as I have to open system/aldente preferences anyway to check if it is enabled or not + I sometimes right click accidentaly and have no way to tell afterwards.

It would be great if the bar icon somehow showed the low power mode state. I was thinking something subtle like a double dash instead of single dash for sailing mode or just classic yellow color like on iPhone.

I am using the AlDente Status icon with sailing mode enabled. macOS Native icon does not show low power mode status either and macOS Colored does nothing for me on apple silicon in dark mode. It just stays white and looks & behaves exactly like macOS Native one.

**Comment by MatthiasKerbl** [comment]
Hi @mr-kew,

Thank you for your suggestion. We will consider adding some kind of indicator in one of the future updates. For now, you can see the Low Power Mode status in the popover by activating the Electrical Specs in the Stats settings window.

Best,
Matthias

**Reply by mr-kew** [reply]
Thanks, looking forward to it.

Btw, as I had now used the pro features more, I switched to macos colored icon as the design lends itself nicely with the rest of the OS. It got me thinking again about how to best indicate the low power mode with this icon alongside the other colors/modes provided by the app.

As the bar color is already occupied by the other modes and yellow outline would probably not look good with blue/orange/green bar, I think the best bet is some sort of separate indicator to the right of the battery icon. Maybe just a dot, similar to the indicator next to the control center icon when facetime is being used.
<img width="38" alt="image" src="https://github.com/AppHouseKitchen/AlDente-Charge-Limiter/assets/26764516/673545bd-3d69-4953-9622-eb20a08eb8e9">

Of course it's just an idea, I believe you'll do a great job anyway, like in the rest of the app.

**Comment by mr-kew** [comment]
It has just been released. Thanks

**Comment by jearley-captech** [comment]
OpenAPI Specification Update

The OpenAPI spec has been exported to build/openapi.yaml and build/openapi.json.

Note: Territory assignment endpoints from placefinder-territory module are not in the generated spec due to how SpringDoc handles Profile-conditional controllers. These endpoints ARE active when running with territory profile.

Territory Assignment Endpoints (manual reference):
- GET /v1/integrations/territory/{id}/assignments - List assignments
- POST /v1/integrations/territory/{id}/assignments - Create assignment
- PUT /v1/integrations/territory/{id}/assignments/{assignmentId} - Update
- DELETE /v1/integrations/territory/{id}/assignments/{assignmentId} - Delete
- GET /v1/integrations/territory/my-territories - Get user territories
- GET /v1/integrations/territory/unassigned - Get unassigned (admin)
- POST /v1/integrations/territory/assignments/bulk - Bulk create (admin)

To view complete API with all endpoints, run the app with all profiles and access Swagger UI at http://localhost:8080/api/swagger-ui/index.html
